src/components/GNB/index.tsx

```diff
@@ -364,17 +365,19 @@ export const GNB: React.FC<GNBProps> = React.memo((props: GNBProps) => {
             <div css={logoAndSearchBox}>
               <LogoWrapper>
                 <li>
-                  <a
-                    href={isPartials ? `${process.env.NEXT_PUBLIC_ACCOUNT_HOST}/` : '/'}
+                  <HomeLink
+                    isPartials={isPartials}
                     aria-label="리디북스 홈으로 이동"
                     css={css`
                       display: flex;
                       align-items: center;
                     `}
                   >
-                    <RidiLogo css={ridiLogo} />
-                    <span className="a11y">RIDIBOOKS</span>
-                  </a>
+                    <a>
```

Comment:
```suggestion
                    <a aria-label="리디북스 홈으로 이동">
```
요기에 붙어야 하지 않을까 싶습니닷..

Reply:
🙏